Bottom: 10ccffc98739146f76f4a83da769fdb55e61f0f8
Top:    0822b1247832f6d88d7d6f78879c93bd511bf4cf
Author: Markus Hitter <mah@jump-ing.de>
Date:   2011-01-23 23:45:38 +0100

Remove all date, user and host comments on all test and sample files.

These items are no longer written, so sample files shouldn't
contain them either. Usually, this was line 2...4.

Reviewed-by: Peter Clifton <pcjc2@cam.ac.uk>
Affects-bug: lp-703914


---

diff --git a/doc/gs/fb-blinker.pcb b/doc/gs/fb-blinker.pcb
index 26066ef..787704a 100644
--- a/doc/gs/fb-blinker.pcb
+++ b/doc/gs/fb-blinker.pcb
@@ -1,7 +1,4 @@
 # release: gpcb 1.99w
-# date:    Sun Mar 16 03:12:26 2008
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/fb-led-5.pcb b/doc/gs/fb-led-5.pcb
index f9d6ac9..bb59791 100644
--- a/doc/gs/fb-led-5.pcb
+++ b/doc/gs/fb-led-5.pcb
@@ -1,7 +1,4 @@
 # release: gpcb 1.99w
-# date:    Fri Dec  7 23:20:12 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/fb-led.pcb b/doc/gs/fb-led.pcb
index c67e652..74f8e9d 100644
--- a/doc/gs/fb-led.pcb
+++ b/doc/gs/fb-led.pcb
@@ -1,7 +1,4 @@
 # release: gpcb 1.99w
-# date:    Wed Dec 26 17:46:01 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/fb-smt.pcb b/doc/gs/fb-smt.pcb
index 2b224b0..4e6f0c9 100644
--- a/doc/gs/fb-smt.pcb
+++ b/doc/gs/fb-smt.pcb
@@ -1,7 +1,4 @@
 # release: gpcb 1.99w
-# date:    Sat Aug  2 23:54:58 2008
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-annulus-1.pcb b/doc/gs/term-annulus-1.pcb
index d3a36ed..51ab107 100644
--- a/doc/gs/term-annulus-1.pcb
+++ b/doc/gs/term-annulus-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:24:16 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-clearance-1.pcb b/doc/gs/term-clearance-1.pcb
index 732df82..21cdef0 100644
--- a/doc/gs/term-clearance-1.pcb
+++ b/doc/gs/term-clearance-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:37:50 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-element-1.pcb b/doc/gs/term-element-1.pcb
index ce5579d..e3ce630 100644
--- a/doc/gs/term-element-1.pcb
+++ b/doc/gs/term-element-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:46:21 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-pad-1.pcb b/doc/gs/term-pad-1.pcb
index c06b12f..e0b1f7f 100644
--- a/doc/gs/term-pad-1.pcb
+++ b/doc/gs/term-pad-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:55:56 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-pin-1.pcb b/doc/gs/term-pin-1.pcb
index a291e3b..ff589fa 100644
--- a/doc/gs/term-pin-1.pcb
+++ b/doc/gs/term-pin-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:57:02 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-platedhole-1.pcb b/doc/gs/term-platedhole-1.pcb
index d1f4a82..a4289b3 100644
--- a/doc/gs/term-platedhole-1.pcb
+++ b/doc/gs/term-platedhole-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 01:29:49 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-tented-1.pcb b/doc/gs/term-tented-1.pcb
index 1abeab2..4af338a 100644
--- a/doc/gs/term-tented-1.pcb
+++ b/doc/gs/term-tented-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 01:25:16 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-thermal-1.pcb b/doc/gs/term-thermal-1.pcb
index 8921a76..37f69a2 100644
--- a/doc/gs/term-thermal-1.pcb
+++ b/doc/gs/term-thermal-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sat Dec  1 22:42:34 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/gs/term-thickness-1.pcb b/doc/gs/term-thickness-1.pcb
index 3bec124..ada246c 100644
--- a/doc/gs/term-thickness-1.pcb
+++ b/doc/gs/term-thickness-1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99w
-# date:    Sun Dec  2 00:40:19 2007
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/doc/pad.pcb b/doc/pad.pcb
index 31218bc..bea2ec2 100644
--- a/doc/pad.pcb
+++ b/doc/pad.pcb
@@ -1,6 +1,4 @@
 # release: pcb-bin 1.99q
-# date:    Tue Mar 21 18:28:39 2006
-# user:    mcmahill (Dan McMahill)
 
 PCB["pad" 600000 500000]
 
diff --git a/doc/puller.pcb b/doc/puller.pcb
index 016ad72..3cd03d5 100644
--- a/doc/puller.pcb
+++ b/doc/puller.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99s
-# date:    Fri Apr 14 23:17:49 2006
-# user:    dj (DJ Delorie)
-# host:    envy.delorie.com
 
 PCB["" 110000 30000]
 
diff --git a/doc/thermal.pcb b/doc/thermal.pcb
index bce2f09..82d0afa 100644
--- a/doc/thermal.pcb
+++ b/doc/thermal.pcb
@@ -1,7 +1,4 @@
 # release: pcb-bin 1.99q
-# date:    Fri Mar 24 21:29:02 2006
-# user:    Dan
-# host:    none
 
 PCB["thermal" 300000 300000]
 
diff --git a/example/LED.pcb b/example/LED.pcb
index b3da6f4..a4da6cc 100755
--- a/example/LED.pcb
+++ b/example/LED.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.7.0.ALPHA
-# date:    Wed Apr 11 11:54:53 2001
-# user:    root (root)
-# host:    tse-eatonha2
 
 PCB("" 3500 3300)
 
diff --git a/example/LED2.pcb b/example/LED2.pcb
index 63754b5..7585a8b 100644
--- a/example/LED2.pcb
+++ b/example/LED2.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.9.9d.ALPHA
-# date:    Sun Jan 20 18:53:49 2002
-# user:    root (root)
-# host:    localhost
 
 PCB("" 3500 3300)
 
diff --git a/tests/inputs/bom_general.pcb b/tests/inputs/bom_general.pcb
index d61b301..d11ffe8 100644
--- a/tests/inputs/bom_general.pcb
+++ b/tests/inputs/bom_general.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99y
-# date:    Wed Jun 17 10:41:24 2009
-# user:    dan (Dan McMahill,At Home ,258-8142,)
-# host:    bondage
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/tests/inputs/gcode_oneline.pcb b/tests/inputs/gcode_oneline.pcb
index c20eacf..6788491 100644
--- a/tests/inputs/gcode_oneline.pcb
+++ b/tests/inputs/gcode_oneline.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99y
-# date:    Wed Jun 24 12:26:14 2009
-# user:    dan (Dan McMahill,At Home ,258-8142,)
-# host:    bondage
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/tests/inputs/gerber_oneline.pcb b/tests/inputs/gerber_oneline.pcb
index c20eacf..6788491 100644
--- a/tests/inputs/gerber_oneline.pcb
+++ b/tests/inputs/gerber_oneline.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.99y
-# date:    Wed Jun 24 12:26:14 2009
-# user:    dan (Dan McMahill,At Home ,258-8142,)
-# host:    bondage
 
 # To read pcb files, the pcb version (or the cvs source date) must be >= the file version
 FileVersion[20070407]
diff --git a/tutorial/tut1.pcb b/tutorial/tut1.pcb
index a49095a..d384f52 100644
--- a/tutorial/tut1.pcb
+++ b/tutorial/tut1.pcb
@@ -1,7 +1,4 @@
 # release: pcb 1.9.9d.ALPHA
-# date:    Sun Jan 20 18:58:24 2002
-# user:    root (root)
-# host:    localhost
 
 PCB("" 3500 3300)
